Assets/Scripts/Singletons/PilotAssetsManager.cs

```diff
@@ -55,87 +64,99 @@ private void Awake()
 		}
 	}
 
-	public async void Init()
-	{
-		List<Task> pilotNameTasks = new List<Task>
-		{
-			Task.Factory.StartNew(() => HumanMaleNames = LoadTextPoolAsync(PilotsConstants.humanMaleNamesPath).Result),
-			Task.Factory.StartNew(() => HumanFemaleNames = LoadTextPoolAsync(PilotsConstants.humanFemaleNamesPath).Result),
-			Task.Factory.StartNew(() => HelicidNames = LoadTextPoolAsync(PilotsConstants.helicidNamesPath).Result),
-			Task.Factory.StartNew(() => OshunianNames = LoadTextPoolAsync(PilotsConstants.oshunianNamesPath).Result),
-			Task.Factory.StartNew(() => OshunianTitles = LoadTextPoolAsync(PilotsConstants.oshunianTitlesPath).Result),
-			Task.Factory.StartNew(() => VestaPrefixes = LoadTextPoolAsync(PilotsConstants.vestaPrefixesPath).Result),
-			Task.Factory.StartNew(() => VestaNames = LoadTextPoolAsync(PilotsConstants.vestaNamesPath).Result),
-			Task.Factory.StartNew(() => Likes = LoadTextPoolAsync(PilotsConstants.pilotLikesPath).Result),
-			Task.Factory.StartNew(() => Dislikes = LoadTextPoolAsync(PilotsConstants.pilotDislikesPath).Result)
-		};
-		await Task.WhenAll(pilotNameTasks).ContinueWith(x => PilotsManager.Instance.RandomisePilots());
-	}
-
-	private async Task<string[]> LoadTextPoolAsync(string fileName)
+	public void Init()
     {
-		string textPool = await DataUtils.ReadFileAsync(fileName);
-		return textPool.Split('\n');
+		StartCoroutine(LoadTextAssetsFromBundle());
     }
 
-	// Combine the value returned with an initial, digit, or second portion
-	private string GenerateNamePortion(string[] namePool)
-	{
-		return namePool[random.Next(0, namePool.Length)];
+    private IEnumerator LoadTextAssetsFromBundle()
+    {
+		// Load pilot text asset bundle "asynchronously" and store reference 
+		AssetBundleCreateRequest bundleRequest = AssetBundle.LoadFromFileAsync(

```

Comment:
I wanted to make a util for this but it was annoying having to return IEnumerator and then get the value from the asset request. So I wrapped in one coroutine for now. Can refactor it later if needed